sections/featured-blog.liquid

```diff
@@ -67,18 +95,15 @@
                 style="--animation-order: {{ forloop.index }};"
               {% endif %}
             >
-              {% render 'article-card',
-                blog: section.settings.blog,
-                article: article,
-                media_aspect_ratio: 1.66,
-                show_image: section.settings.show_image,
-                show_date: section.settings.show_date,
-                show_author: section.settings.show_author,
-                show_excerpt: true
-              %}
+              {% render 'blog-placeholder',
```

Comment:
I personally like that you thought to break this out into a snippet and I'm glad you're thinking about things this way, but I'm not sure this instance meets the somewhat strict criteria we use to determine if something should actually be a snippet. 

It's very subjective, but generally if the snippet doesn't save "a lot" of lines of complicated code and/or the snippet isn't likely to be re-used in multiple files it would be more align with liquid principles to to inline the code here instead.

Reply:
Thank you for letting me know, I removed the snippets and pasted the code in the for block :)